synapse/lookup/pe.py

```diff
@@ -105,160 +148,381 @@ def getRsrcTypes():
+    (0x0031, 'ts'),  # TSONGA
+    (0x0431, 'ts-ZA'),  # TSONGA SOUTH AFRICA
+    (0x001F, 'tr'),  # TURKISH
     (0x041F, 'tr-TR'),  # TURKISH_TURKEY
+    (0x0042, 'tk'),  # TURKMEN
     (0x0442, 'tk-TM'),  # TURKMEN_TURKMENISTAN
+    (0x0080, 'ug'),  # UYGHUR
+    (0x0480, 'ug-CN'),  # UIGHUR_PRC
+    (0x0022, 'uk'),  # UKRAINIAN
     (0x0422, 'uk-UA'),  # UKRAINIAN_UKRAINE
+    (0x002E, 'hsb'),  # UPPER SORBIAN
     (0x042E, 'hsb-DE'),  # UPPER_SORBIAN_GERMANY
+    (0x0020, 'ur'),  # URDU
     (0x0820, 'ur-IN'),  # URDU_INDIA
     (0x0420, 'ur-PK'),  # URDU_PAKISTAN
-    (0x0480, 'ug-CN'),  # UIGHUR_PRC
-    (0x0843, 'uz-UZ-Cyrillic'),  # UZBEK_CYRILLIC
-    (0x0443, 'uz-UZ-Latin'),  # UZBEK_LATIN
+    (0x0043, 'uz'),  # UZBEK (LATIN)
+    (0x7C43, 'uz-Latn'),  # UZBEK (LATIN)
+    (0x0443, 'uz-Latn-UZ'),  # UZBEK (LATIN) UZBEKISTAN
+    (0x7843, 'uz-Cyrl'),  # UZBEK (CYRILLIC)
+    (0x0843, 'uz-Cyrl-UZ'),  # UZBEK (CYRILLIC) UZBEKISTAN
     (0x0803, 'ca-ES-Valencia'),  # VALENCIAN_VALENCIA
+    (0x0033, 've'),  # VENDA
+    (0x0433, 've-ZA'),  # VENDA SOUTH AFRICA
+    (0x002A, 'vi'),  # VIETNAMESE
     (0x042A, 'vi-VN'),  # VIETNAMESE_VIETNAM
+    (0x0052, 'cy'),  # WELSH
     (0x0452, 'cy-GB'),  # WELSH_UNITED_KINGDOM
+    (0x0088, 'wo'),  # WOLOF
     (0x0488, 'wo-SN'),  # WOLOF_SENEGAL
+    (0x0034, 'xh'),  # XHOSA
+    (0x0434, 'xh-ZA'),  # XHOSA_SOUTH_AFRICA
+    (0x0078, 'ii'),  # YI
     (0x0478, 'ii-CN'),  # YI_PRC
+    (0x003D, "yi"),  # YIDDISH
+    (0x043D, 'yi-001'),  # YIDDISH WORLD
+    (0x006A, 'yo'),  # YORUBA
     (0x046A, 'yo-NG'),  # YORUBA_NIGERIA
+    (0x0035, 'zu'),  # ZULU
+    (0x0435, 'zu-ZA'),  # ZULU_SOUTH_AFRICA
+
+    # See Section 2.2.1 of MS-LCID
+    (0x2000, "custom transient 0x2000"),
+    (0x2400, "custom transient 0x2400"),
+    (0x2800, "custom transient 0x2800"),
+    (0x2C00, "custom transient 0x2C00"),
+    (0x3000, "custom transient 0x3000"),
+    (0x3400, "custom transient 0x3400"),
+    (0x3800, "custom transient 0x3800"),
+    (0x3C00, "custom transient 0x3C00"),
+    (0x4000, "custom transient 0x4000"),
+    (0x4400, "custom transient 0x4400"),
+    (0x4800, "custom transient 0x4800"),
+    (0x4C00, "custom transient 0x4C00"),
+    (0x007B, "undefined and unreserved 0x007B"),
+    (0x007D, "undefined and unreserved 0x007D"),
+    (0x0089, "undefined and unreserved 0x0089"),
+    (0x008A, "undefined and unreserved 0x008A"),
+    (0x008B, "undefined and unreserved 0x008B"),
+    (0x008D, "undefined and unreserved 0x008D"),
+    (0x008E, "undefined and unreserved 0x008E"),
+    (0x008F, "undefined and unreserved 0x008F"),
+    (0x0090, "undefined and unreserved 0x0090"),
+    (0x0827, "undefined and unreserved 0x0827"),
+    (0x2008, "undefined and unreserved 0x2008"),
+    (0xF2EE, "reserved 0xF2EE"),
+    (0xEEEE, "reserved 0xEEEE"),
+
+    (0x048D, "plt-MG"),
+    (0x048E, "zh-yue-HK"),
+    (0x048F, "tdd-Tale-CN"),
+    (0x0490, "khb-Talu-CN"),
+
+    (0x0093, "quc, reserved"),
```

Comment:
These two are odd ducks, because "quc" already has a base language code of 0x86, but in MS-LCID, one table has it as 0x0086 and the other has it as 0x0093.